packages/test/test-end-to-end-tests/src/test/containerRuntime.spec.ts

```diff
@@ -176,7 +176,15 @@ describeCompat(
 		for (const explicitSchemaControl of choices) {
 			for (const compression of choices) {
 				for (const chunking of choices) {
-					it(`test explicitSchemaControl = ${explicitSchemaControl}, compression = ${compression}, chunking = ${chunking}`, async () => {
+					it(`test explicitSchemaControl = ${explicitSchemaControl}, compression = ${compression}, chunking = ${chunking}`, async function () {
+						// Skip this test for FRS and ODSP as its timing is flaky.
+						// This test is covering client logic and the coverage from other drivers/endpoints is sufficient.
+						if (
+							provider.driver.type === "odsp" ||
+							(provider.driver.type === "r11s" && provider.driver.endpointName === "frs")
```

Comment:
Should we just skip for all `r11s` drivers?  I mean, why not?  (am I even right that there's other endpoints for r11s?).

That's at least nice that it hasn't failed for t9s